### DISCUSSION TITLE
Frontend bug in repository contributors page

### DISCUSSION THREAD

**Discussion by Hamaad616** [community/community#32589] [Repositories]
I found a bug on github repository contributors page where the graph glitches on screen sizes
![image](https://user-images.githubusercontent.com/46200983/189498367-5fc48c10-369f-452e-ba91-a827964369e0.png)
I may suggest decreasing font sizes according to breakpoints thank you!

**Comment by melvincwng** [comment]
@Hamaad616 Hi Hamaad, thanks for highlighting this issue to the community.

For bugs, please kindly raise a ticket to [GitHub Bug Report](https://support.github.com/contact/bug-report). Once a ticket is raised, a support engineer at GitHub would look at resolving this issue.

Hope this clarifies and answers your query 😊

**Reply by jsoref** [reply]
@melvincwng: interesting. GitHub [support](https://support.github.com/contact/bug-report) has been generally pushing me to file items via [community/discussions](https://github.com/community/community/discussions).

I was just browsing this category before deciding which to use. It isn't particularly obvious, it's worse than throwing darts, and given how large discussions are, it's fairly expensive. (Admittedly, my list of [bug reports filed](https://support.github.com/tickets/personal/0) is multiple pages long...)

**Reply by melvincwng** [reply]
@jsoref Yea, I've noticed in some discussion threads in GitHub Community whereby the support team has requested to file items via the discussions platform.

Not sure what is happening since GitHub Support should be the main avenue for filing bugs.
